README.md

```diff
@@ -44,3 +44,8 @@ To recompile everything, simply run grunt from inside the container:
 ```
 docker-compose run --rm node grunt
 ```
+
+## Hosting
+Timelord is currently hosted with Kubernetes at Google Cloud Platform in the project: `reload-internal-alpha` <br/>
```

Comment:
I'm using it specifically in the README to create newlines -- admitted: It was a quick google, but it works!
When I separated each line in my editor with return, the README output would be concatenated. 
Is there a better way to do this?

Reply:
Fixed with white spacing instead :+1:

Reply:
This is a limitation of your IDE. VSCode and other editors let your hard wrap the text paragraphs with a shortcut, thus inserting line breaks at 80 chars (which is just the norm, there is no standard for Markdown). You could also just not do line breaks at all and just use soft wrapping in the IDE instead. It will still render correctly wherever it is shown.

Double line breaks create a new paragraph, which is also useful.

But as I said - it would be useful to enable a Markdown linter so we have a unified syntax everywhere.